src/components/dictionaryConcepts/components/CreateConceptForm.jsx

```diff
@@ -52,47 +52,47 @@ const CreateConceptForm = props => (
           <div className="form-group col-md-5">
             <label htmlFor="class">Class</label>
             <div>
-              { (props.concept.toString().trim() === 'symptom-finding') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control symptom-finding"
-                  required
-                  onChange={props.handleChange}
-                >
-                  <option key="Symptom-Finding">Symptom-Finding</option>
-                  <option key="Symptom">Symptom</option>
-                  <option key="Finding">Finding</option>
-
-                </select>
-              </React.Fragment>
+              {(props.concept.toString().trim() === 'symptom-finding') &&
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
+                    value={props.state.concept_class}
+                    className="form-control symptom-finding"
+                    required
+                    onChange={props.handleChange}
+                  >
+                    <option key="Symptom-Finding">Symptom-Finding</option>
+                    <option key="Symptom">Symptom</option>
+                    <option key="Finding">Finding</option>
+                  </select>
+                </React.Fragment>
               }
-
-              { (props.concept.toString().trim() === 'set') &&
-              <React.Fragment>
-                <select
-                  id="class"
-                  name="concept_class"
-                  value={props.state.concept_class}
-                  className="form-control set"
-                  required
-                  onChange={props.handleChange}
-                >
-                  <option key="LabSet">LabSet</option>
-                  <option key="ConvSet">ConvSet</option>
-                  <option key="MedSet">MedSet</option>
-
-                </select>
-              </React.Fragment>
+              {(props.concept.toString().trim() === 'set') &&
+                <React.Fragment>
+                  <select
+                    id="class"
+                    name="concept_class"
+                    value={props.state.concept_class}
```

Comment:
Same here

Reply:
Let me revert all the formatting  changes and I will create a separate ticket  for it